Neolution.Utilities/StringExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Neolution.Utilities;
```

Comment:
@neoscie I wasn't sure where to place this extension, should I move it to a dedicated folder or it's ok to keep it here?

Reply:
string is not a primitive type.
<img width="1127" height="195" alt="image" src="https://github.com/user-attachments/assets/7f6d842f-c76d-496b-a8f2-8592c88e50a2" />

Should I move it anyway to the Text namespace/folder?

I thought that using this library in a project would be like adding the single global using "Neolution.Utilities" (and few others maybe) to avoid having too many different usings just for different extensions.